apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -25,6 +25,13 @@ const ERRORS = {
         messageForUser: 'api.meterReading.METER_READING_DATE_IN_FUTURE',
         messageInterpolation: { givenDate },
     }),
+    METER_READING_WRONG_VALUES_COUNT: (meterNumber, numberOfTariffs, fieldsNames) => ({
```

Comment:
A bit unusual format....Was it used before?

Reply:
Yes, this is not first using. Id like to pass some data to error to make it more informative.